_posts/2023-07-15-gams-blog.md

```diff
@@ -0,0 +1,430 @@
+---
+layout: post
+title: "JuMP, GAMS, and the IJKLM model"
+date: 2023-07-15
+---
+
+A [recent blog post](https://www.gams.com/blog/2023/07/performance-in-optimization-models-a-comparative-analysis-of-gams-pyomo-gurobipy-and-jump)
+by GAMS demonstrated a significant performance difference between JuMP and
+GAMS on a model they call IJKLM. The purpose of this blog post is to explain
```

Comment:
Suggested edits:
A recent blog post [...] on a model they call IJKLM. We respond to this blog post by explaining the difference in performance and presenting an alternative JuMP implementation with asymptotically better performance. We also identify that differences in the input data format—not anything intrinsic to the respective libraries—explain the performance difference between the Pyomo model and the JuMP model. Finally, we respond to general claims about domain specific languages and the trade-offs of developing optimization models within a fully-featured programming language like Julia or Python.